### PR TITLE
Update Runtime(s) to 23.08

### PR DESCRIPTION
**Pull request by Justinzobel**

### CODE REVIEW

**Comment by Justinzobel** [comment]
bot, build

**Comment by Erick555** [comment]
this needs patch for ffmpeg: https://github.com/flathub/com.spotify.Client/blob/master/ffmpeg-binutils-2.41.patch

**Comment by zWhdmB5T** [comment]
'ERROR   src.manifest: Failed to check archive exiv2/v0.28.1.tar.gz with AnityaChecker: Error downloading upstream source: 404, message='Not Found', url=URL('https://github.com/Exiv2/exiv2/releases/download/v0.28.1/exiv2-0.28.1-Source.tar.gz')`

It seems to be <https://github.com/Exiv2/exiv2/archive/refs/tags/v0.28.1.tar.gz> (but <https://github.com/Exiv2/exiv2/releases/download/v0.28.1/exiv2-0.28.1-Linux64.tar.gz> e.g.)

File: org.kde.digikam.json [hunk 1]
```diff
@@ -272,8 +272,8 @@
                     "sources": [
                         {
                             "type": "archive",
-                            "url": "https://github.com/Exiv2/exiv2/releases/download/v0.27.6/exiv2-0.27.6-Source.tar.gz",
-                            "sha256": "4c192483a1125dc59a3d70b30d30d32edace9e14adf52802d2f853abf72db8a6",
+                            "url": "https://github.com/Exiv2/exiv2/archive/refs/tags/v0.28.1.tar.gz",
```
Comment (hfiguiere) [inline org.kde.digikam.json:275]: 0.28 requires an extra dependency and actually break API. Did you really test this?

Comment (zWhdmB5T) [reply org.kde.digikam.json:275]: I'm not really into it…, sorry.

**Comment by zWhdmB5T** [comment]
Thank you!

It works.

**Pull request closed without merging**